.github/workflows/test.yml

```diff
@@ -12,9 +12,25 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  check_formatting:
+    name: Check formatting of files
+    runs-on: ubuntu-latest
+    container:
+      image: nycplanning/build-geosupport:latest
```

Comment:
use base? There's not a huge difference in size but can use a smaller image

Actually, if we just want a very small one that certainly will work https://hub.docker.com/r/pyfound/black/tags

Reply:
Looks like there's also an official gha
https://black.readthedocs.io/en/stable/integrations/github_actions.html

Reply:
I'll switch to the base image. I think I prefer using our container + CLI command rather than a marketplace action. looks like in both cases you end up using CLI arguments and using the later seems to add a layer of separation/abstractions without much benefit

Reply:
Also the argument (in our image vs black image) that we might add more to this job later. I still like the marketplace action, but lgtm

Reply:
Yeah, as long as our base image is small enough to run quickly. I'm guessing the GHA would be much quicker